platform/firecamp-platform/src/components/activity-bar/explorer/treeItemRenderer.tsx

```diff
@@ -34,21 +34,33 @@ const FolderCloseIcon = (props) => (
 export default {
   renderItemArrow: ({ item, context }) => {
     // console.log(item, 'arrow context');
+    const leftAligned : boolean = false;
+
+    const alignClass = cx({
+      "text-left": leftAligned,
+      "text-right": !leftAligned,
+    });
+    const requestTokenClass = cx({
+     "pr-5": leftAligned,
```

Comment:
these classes not begin generated by tailwind.

Reply:
try `pnpm build:tailwind` at the `platform/firecamp-ui` directory